src/markdown-pages/opentelemetry-masterclass/hands-on/traces.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'View your traces'
+template: 'GuideTemplate'
+description: 'View your traces in New Relic'
+procIdx: 3
+---
+
+<Callout variant="course">
+
+This lesson is a part of our OpenTelemetry masterclass. If you haven't already, checkout the [chapter introduction](/opentelemetry-masterclass/hands-on).
+
+Each lesson in this lab builds on the last one, so make sure you [view a summary of your data](/opentelemetry-masterclass/hands-on/data-summary) before proceeding with this one.
+
+</Callout>
+
+Now that you've seen a summary of your OpenTelemetry data in New Relic, take a closer look at your traces.
+
+<Steps>
+
+<Step>
+
+From the left-hand navigation, select **Distributed tracing**:
+
+![Distributed tracing](../../../images/opentelemetry-masterclass/distributed-tracing.png)
+
+Here, you see some general information about traces in your service. You see the number of traces you receive over time, the duration of those traces, and the number of traces with errors. In the table at the bottom, you see [trace groups](https://docs.newrelic.com/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui#trace-groups). These organize your traces, grouping them by their root span.
+
+</Step>
+
+<Step>
+
+Click the **HTTP GET** trace group:
+
+![Trace group](../../../images/opentelemetry-masterclass/dt-row.png)
+
+</Step>
+
+<Step>
+
+At the top, you see traces plotted on a timeline:
+
+![Plotted traces](../../../images/opentelemetry-masterclass/plotted-traces.png)
+
+</Step>
+
+<Step>
+
+At the bottom, you see a chart with data for each of the traces:
+
+![Charted traces](../../../images/opentelemetry-masterclass/charted-traces.png)
+
+Every row in this chart includes the root span’s name, the start time, and duration. It also indicates the number of spans in the trace that have an error status.
+
+</Step>
+
+<Step>
+
+Order the table by ascending number of errors and click one of the top traces:
+
+![Ascending errors](../../../images/opentelemetry-masterclass/ascending-errors.png)
+
+For now, you’re going to view a trace with no errors. You’ll investigate traces with errors in the next lesson.
+
+In this new view, you see the timeline of the spans in your trace:
+
+![Trace timeline](../../../images/opentelemetry-masterclass/trace-timeline.png)
+
+There are two blue blocks, one for each span. The blue color means that the span does not have an error status.
+
+The span on top is the root span. It’s the one named “HTTP GET”:
+
+![Trace root span](../../../images/opentelemetry-masterclass/dt-success-root.png)
+
+The bottom span is an **in-process** span. In New Relic, this means that it represents an internal function call, and that it's neither an exit nor entry span for the service:
+
+![Trace in-process span](../../../images/opentelemetry-masterclass/dt-success-in-process.png)
+
+In the chart below the timeline, you only see the root span.
```

Comment:
It's less obvious in this particular trace, but the small blue block is under the big blue block on the left side. It's under because it happens logically after the start of the top one. It's to the right because it happens temporally after the start of the top one. I'll try to capture a better image or highlight this in some way. I think this is partially a UI concern.